tests/Restore-DbaDatabase.Tests.ps1

```diff
@@ -683,6 +683,24 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
         }
     }
 
+    Context "Test recovery via exported backup history" {
+        $DatabaseName = 'rectest'
+        Get-DbaDbBackupHistory -SqlInstance $script:instance2 -Database $DatabaseName -LastFull | Export-Clixml -Depth 5 -Path "C:\temp\${DatabaseName}.xml"
+        $results = Get-DbaBackupInformation -Import -Path "C:\temp\${DatabaseName}.xml" | Restore-DbaDatabase -SqlInstance $script:instance2 -TrustDbBackupHistory
```

Comment:
We do not use `Export-Clixml`. `-Export` parameter should be used with `Get-DbaDbBackupHistory` if you want to contribute a test update.

Use of hardcoded file paths is discouraged and not used in our tests. You can review other tests that are generating temp files to see what that use but it should be a path that is available in Windows or Linux (don't test on this OS yet but may be in the future). Use of `$env:TEMP` is a preferred path but I can't recall what others are using right off.

Reply:
Example 2 of the docs for the Backup information command would be a scenario to include in testing

https://docs.dbatools.io/Get-DbaBackupInformation

Reply:
Is there a way to get the export created directly from Get-DbaDbBackupHistory?  The goal is to avoid the header scans of over 2000 files (mostly logs) in our backup directory, which is what Get-DbaBackupInformation would do if we let it.

Reply:
Also, I checked the existing tests and found C:\temp in multiple places.  We thought about the linux use case and were suprised to see it, but my goal was to use what was done previously as a model.